### ISSUE TITLE
Working with Rules: Unexpected large font in SourceCode documentation

### ISSUE THREAD

**Issue by platinumazure** [eslint/eslint.github.io#491]
In the [SourceCode section of "Working with Rules"](https://eslint.org/docs/developer-guide/working-with-rules#contextgetsourcecode), there's a section in which the different range/skip/count options are documented for some of the newer SourceCode methods. That section is rendered with a larger font than the surrounding content.

I opened the issue up here because I assumed this might be a CSS problem. If it's a problem with the source content (i.e., the markdown), let me know and I can move this to eslint/eslint.

**Comment by AlokTakshak** [comment]
@platinumazure 
![image](https://user-images.githubusercontent.com/26347838/57285619-36b7c600-70d1-11e9-90af-90c87586303b.png)
as we can see  blockquote is providing the font size 21.25px;

**Comment by AlokTakshak** [comment]
@platinumazure 
It is in styles/main.css
should i update it and raise a PR?

**Comment by platinumazure** [comment]
@AlokTakshak Nice find! Feel free to raise a PR, but be careful-- I think we only really need the code samples to use a smaller font size. Block quotes for actual prose should probably still be larger size. So I would recommend creating a more specific selector to override for code samples, if at all possible. Thanks for taking a look!

**Comment by AlokTakshak** [comment]
@platinumazure 
`blockquote p, ul {
    font-size: 17px;
}`
I think this should be fine for now or we need to give  for all elements inside blockquote
`blockquote *  {
    font-size: 17px;
}`

**Comment by btmills** [comment]
https://github.com/eslint/eslint/blob/9590587cef74c936ef9b7ce2d22a71e2fd0fbbc4/docs/developer-guide/working-with-rules.md#L415-L429

After taking a look at this, my opinion is that this is an issue with the Markdown source and not the CSS. That section is written as a blockquote with the `>` prefixes, but it seems that was being used for indentation rather than quote-like semantics. If we don't need them to be a blockquote, removing the `>` prefix from those lines would fix this. What do you all think?

**Comment by AlokTakshak** [comment]
@btmills Yeah that will also work, do you want me to remove the `>` prefix?

**Comment by platinumazure** [comment]
@AlokTakshak Let's try that first and see how everything looks. If we need to do any further changes, we could do them either in the same PR or in a separate PR. Thanks!

@btmills Please speak up if you feel differently.

**Comment by AlokTakshak** [comment]
> @AlokTakshak Let's try that first and see how everything looks. If we need to do any further changes, we could do them either in the same PR or in a separate PR. Thanks!
> 
> @btmills Please speak up if you feel differently.

so we are going with initial (i.e adding css property) for now?

**Comment by platinumazure** [comment]
@AlokTakshak Sorry for being unclear. I meant we should change the Markdown source first since it's semantically incorrect. Let's fix that first, then we can decide if some CSS change might be needed later. Thanks!

**Comment by AlokTakshak** [comment]
> @AlokTakshak Sorry for being unclear. I meant we should change the Markdown source first since it's semantically incorrect. Let's fix that first, then we can decide if some CSS change might be needed later. Thanks!

ok, i will make changes in same PR

**Comment by AlokTakshak** [comment]
@platinumazure   I have removed the '>' prefix, you can check the PR
